CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+## [TBD] - TBD
+* update MSIDBaseBrokerOperationRequest in common-core
```

Comment:
This PR seems more about fixing typos in comments. Should we include that in changelog instead?

Reply:
Sure, I will update the changelog